work11/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+const wrap = document.getElementById("js-wrap")
+const ul = document.getElementById("js-list");
+// const url = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// const url = "https://myjson.dit.upm.es/api/bins/bu5z";
+const url = "https://myjson.dit.upm.es/api/bins/ajy3"
+
+
+async function getData() {
+  loading();
+  try {
+    const response = await fetch(url);
+    if (response.ok) {
+      const data = await response.json();
+      return data;
+    } else {
+      throw alert("apiデータが取得できません")
+    }
+  } catch (e) {
+    wrap.textContent = e.message;
+  } finally {
+    hideLoading();
+  }
+}
+
+getData().then((value) => {
+  console.log({ value })
+  console.log(value.data)
+  if (!value.data.length) {
+    throw alert("配列空です。")
```

Comment:
Q
Here, too.
Why did they go  throw with the alert?

Reply:
https://github.com/yasu-pro/js-handson/pull/11#discussion_r777754531
Including the above URL

The reason I used alert () was because I thought it was easy to see visually.

This time, I changed throw.

However, when handling exceptions,
`throw new Error ();`
Is it better to use

commit : bd55b98